docs/examples/getting-started/quickstart.py

```diff
@@ -23,14 +23,20 @@
 postgres = Postgres.get(name="my_rdbms")
 warehouse = Snowflake.get(name="my_warehouse")
 kafka = Kafka.get(name="my_kafka")
+webhook = Webhook(name="fennel_webhook")
 
 
 # /docsnip
 
 
 # docsnip datasets
 @dataset
-@source(postgres.table("product_info", cursor="last_modified"), every="1m")
+@source(
+    postgres.table("product_info", cursor="last_modified"),
+    every="1m",
```

Comment:
can you please remove every field so that we can hopefully still fit the `source` decorator in a single line? If that doesn't work, try shortening the postgres table name from "product_info" to just "product". 

This is the first line of code shown in demo so I really want it to feel neat/simple.